Stresser/FileArtifact.cpp

```diff
@@ -0,0 +1,39 @@
+#include "FileArtifact.h"
+
+FileArtifact::FileArtifact(std::string name, std::string data) : IArtifact(name, data)
+{
+	this->install();
+}
+
+FileArtifact::~FileArtifact()
+{
+	this->uninstall();
+}
+
+void FileArtifact::install() const
+{
+	std::string fileName = this->data;
+
+	AutoHandle hFile(CreateFileA(
+		fileName.c_str(),		// Open file
+		GENERIC_WRITE, 			// Open for writing
+		0, 						// Do not share
+		NULL, 					// Default security
+		OPEN_ALWAYS, 			// Open or create
+		FILE_ATTRIBUTE_NORMAL, 	// Normal file
+		NULL
+	));
+
+	if (!hFile.get())
```

Comment:
We prefer to check the expected value, such: `INVALID_HANDLE_VALUE` or `nullptr` instead of writing (!)
for better clean code.